examples/websub-hub-client-sample/publisher.bal

```diff
@@ -10,7 +10,7 @@ public function main() {
     // Starts the internal Ballerina Hub.
     io:println("Starting up the Ballerina Hub Service");
 
-    var result = websub:startHub(new http:Listener(9191));
+    var result = websub:startHub(new http:Listener(9191), "/websub", "/hub");
```

Comment:
What is the requirement of basePath? Can we default it to "/" ? IMO the resource path would be enough as we are not running multiple services on particular port. And also it looks like we are asking too much redundant path(basePath and path) info. WDYT?

Reply:
These are optional parameters. Added these just for clarity of the BBE, but we could maybe do with minimum args, will update.

But, there could be scenarios where the same listener is used by the hub service and some other service, IMO.

Reply:
Other services means you need to attach them later to the same listener ryt? If so can't they have some different basePaths? I'm wondering whether we can reduce these path asks from the user. Providing two paths as two variable is pretty rare.

Reply:
They can, but what if the user wants to use "/" for the other service and something else for this? 

We need the resource paths since there are two resources now.

IMO, since we have default values anyway this is OK? In case the user wants to change it he still has the option.